### PR TITLE
create skeleton

### PR DESCRIPTION
**Pull request by rudigiesler**

### CODE REVIEW

**Comment by hodgestar** [comment]
:+1:

I'm not sure we need all the building-a-single-file grunt tasks from the skeleton because this will be a library, but it's easy enough to strip out later.

File: utils/test.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash -e
+jshint `find lib test examples -name "*.js"`
+mocha -R spec `find test examples/test -name "*.js"`
```
Comment (justinvdm) [inline utils/test.sh]: I don't think we need these if we are using grunt.

**Comment by justinvdm** [comment]
Left a minor comment. Seconding us probably not needing the build files and tasks, might be worth removing before this lands, but your call.

File: utils/test.sh (hunk 1 above)
Comment (rudigiesler) [inline utils/test.sh]: This is what is used in `package.json`, which I think Travis will use? Should we take it out and replace with `grunt test` in `package.json`?

**Comment by rudigiesler** [comment]
@hodgestar I thought I removed all the build/concat things from the Grunt file. Could you show me where I missed?

File: utils/test.sh (hunk 1 above)
Comment (hodgestar) [inline utils/test.sh]: +1 on using grunt test in package.json.

Comment (justinvdm) [inline utils/test.sh]: Yup :)

**Comment by hodgestar** [comment]
@rudigiesler Ah, I see you did. What confused me is that `path.src.app` (and `prd` and `all`) are still around. I would think we would only really need `all` now and that `app` and `prd` can go? Maybe `all` can then be just directly on `path.src`.

If we're going to do these sort of library-repos rather than app-repos, maybe we need to think about how to support skeletons for both (not for this PR :).

**Comment by justinvdm** [comment]
:+1: once travis is happy

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**